### ISSUE TITLE
Allow sorting by name, average rating or number of reviews

### ISSUE THREAD

**Issue by ewangi** [linuxmint/mintinstall#223]
Currently the software packages in the Software Manager categories are ordered, roughly speaking, by the number of reviews (rating might also be a factor as this order is not set in stone - for example, the System Tools category currently lists the packages in the order 238 reviews, 165, 160, 207, 174, etc.).

For reasons of flexibility and control, it could be useful to add header buttons to also allow the user to order the packages alphabetically (ascending), by rating (descending), or by the number of reviews (descending).

**Comment by xenopeek** [comment]
Packages are currently sorted by their points first, name second. Points are calculated as the sum of the review score minus 3 of each review (e.g., a package with 2 reviews with score 4 and 5 would have 3 points).

**Comment by haggen88** [comment]
I agree ... it would be very useful to add a button to organize the applications alphabetically.
